asyncapi/asb/Package.md

```diff
@@ -0,0 +1,23 @@
+Listen to [Microsoft Azure Service Bus](https://docs.microsoft.com/en-us/java/api/overview/azure/servicebus/client?view=azure-java-stable&preserve-view=true) from Ballerina.
+
+## Package overview
+
+The `ballerinax/trigger.asb` is a [Ballerina](https://ballerina.io/) trigger for Microsoft Azure Service Bus.
+This package provides the capability to access Microsoft Azure Service Bus messages from Service Bus queues, topics, and subscriptions, etc.
+
+### Compatibility
+
+|                            | Version               |
+|----------------------------|-----------------------|
+| Ballerina Language         | Swan Lake Beta3       |
+| Azure Service Bus SDK      | 3.5.1                 |
+
+## Report issues
+
+To report bugs, request new features, start new discussions, view project boards, etc., go to the [Ballerina Extended Library repository](https://github.com/ballerina-platform/ballerina-extended-library)
```

Comment:
Let's check if we have to create issues here or in https://github.com/ballerina-platform/asyncapi-triggers

Reply:
We have followed the same structure as in the generated code https://github.com/ballerina-platform/asyncapi-triggers/blob/main/asyncapi/slack/Package.md. I will recheck this again & do the update if necessary. Thank you.